os/network-config-with-networkd.md

```diff
@@ -86,6 +86,29 @@ DHCP=yes
 
 To apply the configuration, run `sudo systemctl restart systemd-networkd`. Check the status with `systemctl status systemd-networkd` and read the full log with `journalctl -u systemd-networkd`.
 
+## Debug networkd
+
+If you've faced some problems with networkd you can enable debug mode following the instructions below.
+
+```sh
+mkdir -p /etc/systemd/system/systemd-networkd.service.d/
+```
+
+Create `/etc/systemd/system/systemd-networkd.service.d/10-debug.conf` with following content:
+
+```sh
+[Service]
+Environment=SYSTEMD_LOG_LEVEL=debug
+```
+
+And restart systemd-networkd service:
+
+```sh
+systemctl daemon-reload
+systemctl restart systemd-networkd
+journalctl -u systemd-networkd
```

Comment:
i think something like `journalctl -b -u systemd-networkd` makes more sense here. no use in fetching all of the logs, but logs since boot should suffice for this example.